### DISCUSSION TITLE
How to open a scheme relative URI in an Ionic Capacitor iOS app

### DISCUSSION THREAD

**Discussion by ghost** [ionic-team/capacitor#4251] [Q&A]
In an Ionic/Capacitor iOS app, I need to use an external library; the main script (_jwplayer.is_  ,in my case)  requests other external JS modules through a scheme relative URI (eg: **_//ssl.p.jwpcdn.com/player/v/8.19.0/jwpsrv.js_**).

While in the Android version is possible to set 'androidScheme': 'https' in the capacitor.config.json, this is not allowed on iOS. 
So that the request become **_capacitor://ssl.p.jwpcdn.com/player/v/8.19.0/jwpsrv.js_** and of course it fails because the resource cannot be found.

At the moment, instead of loading the script from CDN, I have embedded it in the asset folder and changed the URI to 'https://... ' from '//...', but I have still problems because on of the requested module makes other requests   using a schema relative URI.

Is there any way to force the protocol to 'https'? .
Thanks for any idea.

**Comment by cdills** [comment]
I'm running into this as well. Was a solution or workaround ever found?

**Reply by ghost** [reply]
Sorry but I haven't find a way to force the set the scheme/protocol to https. 
I had to embed the script into my project and manually change all the refs to '//....'

**Reply by davegroff** [reply]
I am running into the same problem now several years later. Does anyone have any other solutions or idea?
